proto/src/determined/resourcepool/v1/resourcepool.proto

```diff
@@ -171,7 +171,7 @@ message ResourcePool {
   string agent_docker_runtime = 26;
   // The docker image to use for the agent when using dynamic agents
   string agent_docker_image = 27;
-  // The Fluent docker image to use
+  // Deprecated, the Fluent docker image to use
   string agent_fluent_image = 28;
```

Comment:
you can use `[deprecated = true]` to mark this. should be able to grep proto to find an example.